cloudify_cli/commands/audit_log.py

```diff
@@ -22,6 +22,12 @@ def auditlog():
               help=helptexts.AUDIT_CREATOR_NAME)
 @click.option('-e', '--execution-id',
               help=helptexts.AUDIT_EXECUTION_ID)
+@click.option('-i', '--since',
+              help=helptexts.AUDIT_SINCE)
+@click.option('-f', '--follow', '--tail',
```

Comment:
Can we maybe not support --tail for this so that we can gradually phase out that wrong argument naming?

Reply:
(I mean not support that as the name of the argument- just calling it --follow is correct)

Reply:
sure thing, I wasn't sure if now was the good time for that, I'll gladly drop `--tail` from the list